src/app/beer_garden/systems.py

```diff
@@ -72,7 +72,7 @@ def create_system(system: System) -> System:
             system.max_instances = len(system.instances)
 
     if system.namespace is None:
-        system.namespace = beer_garden.config.get("namespaces.name")
+        system.namespace = beer_garden.config.get("namespaces.local")
```

Comment:
Should this be using the garden name instead? I think we should get rid of that whole namespaces config section as it doesn't really make sense anymore.